packages/components/organisms/f-checkout/src/demo/checkout-dinein.json

```diff
@@ -10,8 +10,8 @@
   },
   "fulfilment": {
     "time": {
-      "from": "2020-01-01T00:00+00:00",
-      "to": "2020-01-01T00:00+00:00"
+      "asap": false,
+      "scheduled": null
```

Comment:
Is there a specific reason as to why the dine in mock was changed differently to the other two? I am happy to rectify it after, but this is the only thing that stuck out to me in this PR.

Reply:
Because it's also a valid value (what most users will see when they land on Checkout). I wanted to have different valid values in the files, but there's no specific reason to have this in the `dinein` file. Are you ok to leave it as it is or would you like me to change it to something else?

Reply:
On dine in there is no selector so in the background the API always sets `asap` to true and sends us one `to` and `from` time slot. But as mentioned, I am happy to patch that after. 🙂